### PR TITLE
Remove confusing reference to yarn in GitHub pull request template

### PR DESCRIPTION
**Pull request by sampathBlam**
Update Pull Request template with the correct lint command

### Description

- Replace `yarn lint` with `npm run lint` in the PULL_REQUEST_TEMPLATE.md

### Motivation and Context

To provide correct lint command in the PR template.

### How Has This Been Tested?

Previewed the markdown file in local machine.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project (run `$ yarn lint`).

### CODE REVIEW

File: src/features/todos/components/TodosWebview.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ class TodosWebview extends Component {
     const webViewInstance = this;
     this.webview.addEventListener('dom-ready', () => {
       webViewInstance.webview.setUserAgent(userAgent(true));
-    })
+    });
```
Comment (kytwb) [inline src/features/todos/components/TodosWebview.js:137]: I would remove this change for now 🙏 So this file doesn't get edited in a commit related to updating the GitHub pull request template.

**Comment by kytwb** [comment]
Also let's try to name our pull requests so they are understandable by humans browsing the repository:

Screen Shot 2020-05-08 at 11 15 08 AM

Thank you guys 🙏

File: src/features/todos/components/TodosWebview.js (hunk 1 above)
Comment (sampathBlam) [reply src/features/todos/components/TodosWebview.js:137]: Sure. Makes sense.

**Comment by sampathBlam** [comment]
@kytwb : Removed the lint fix. And I will make sure the pull requests are more readable in future. Can you kindly review the change ?

**Pull request merged**